core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/dot-edit-content-sidebar.component.html

```diff
@@ -56,17 +48,22 @@
 
             <dot-edit-content-sidebar-section [title]="'Workflow' | dm">
                 <dot-edit-content-sidebar-workflow
-                    [isNew]="isNew"
-                    [workflowSchemeOptions]="workflowOptions"
+                    [workflowSelection]="{
+                        schemeOptions: store.workflowSchemeOptions(),
+                        noWorkflowSelectedYet: store.showSelectWorkflowWarning()
+                    }"
                     [(showDialog)]="$showDialog"
                     (onSelectWorkflow)="store.setSelectedWorkflow($event)"
+                    [resetWorkflowAction]="store.getResetWorkflowAction()"
+                    (onResetWorkflow)="fireWorkflowAction($event)"
                     [workflow]="{
```

Comment:
Could the workflow input be computed with all options?

Reply:
good idea.